code/controllers/subsystem/ticker.dm

```diff
@@ -360,35 +360,45 @@ SUBSYSTEM_DEF(ticker)
 		if(is_banned_from(new_player_mob.ckey, list("Captain")))
 			CHECK_TICK
 			continue
+		if(!ishuman(new_player_mob.new_character))
+			CHECK_TICK
```

Comment:
This check tick seems kinda redundant. Same in next loop.

Reply:
While this is looping for new players, this is checking if those new players have a human new character.
I could change it to `if(new_player_mob.new_character)` if you'd rather.
At this point there's only humans and they haven't been turned into silicons, I think, but can't guarantee it without testing.